src/user/guards/is-user-author.guard.ts

```diff
@@ -19,10 +13,7 @@ export class IsUserAuthorGuard implements CanActivate {
     );
 
     if (request.user.id !== recipeFromDb.authorId) {
-      throw new HttpException(
-        'User do not own this recipe',
-        HttpStatus.FORBIDDEN,
-      );
+      return false;
     }
 
     return true;
```

Comment:
you could do it in one line:
`return request.user.id === recipeFromDb?.authorId`